packages/react-components/react-popover/library/src/components/PopoverSurface/usePopoverSurfaceStyles.styles.ts

```diff
@@ -15,18 +15,24 @@ export const arrowHeights: Record<PopoverSize, number> = {
   large: 8,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32055&commitId=980b78399e659b28bc322b54cff7ff69ff8fb2c3&env=prod&clientType=FLUENTUI)


  <details><summary><b>AvatarGroup Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|309| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|313| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|1025| Changed|

</details>


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|1826| Changed|

</details>


  <details><summary><b>Popover Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|3602| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|256| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|3603| Changed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|1091| Changed|
|Popover Converged.when rendering inline, it should not render behind relatively positioned elements.PopoverSurface focused.chromium.png|484| Changed|

</details>


  <details><summary><b>Portal</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Portal.Apply Class Names.should have green border.chromium.png|483| Changed|

</details>


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow - RTL.chromium.png|461| Changed|
|Positioning.arrow.chromium.png|486| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip Converged</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|130| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|8| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|8| Changed|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|1| Changed|
|Tooltip Converged.withArrow.chromium.png|1| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|7| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>